Makefile

```diff
@@ -237,10 +237,18 @@ pre-build:
 
 sed-common-module:
 	@echo 'Replacing all missed entity relative paths to @tbcm/common inside node_modules :: fixes Module.Import errors'
+	
+	# entity.js
 	@if [[ $(shell uname) == "Darwin"* ]]; \
 	then sed -i '' -E 's/( require\("[^,]+\/packages\/common\/dist[^,]+"\).)/ require("@tbcm\/common")./g' ./apps/api/dist/**/**/*.entity.js; \
 	else sed -i -E 's/( require\("[^,]+\/packages\/common\/dist[^,]+"\).)/ require("@tbcm\/common")./g' ./apps/api/dist/**/**/*.entity.js; \
 	fi
+
+	# controller.js
+	@if [[ $(shell uname) == "Darwin"* ]]; \
+	then sed -i '' -E 's/( require\("[^,]+\/packages\/common\/dist[^,]+"\).)/ require("@tbcm\/common")./g' ./apps/api/dist/**/*.controller.js; \
+	else sed -i -E 's/( require\("[^,]+\/packages\/common\/dist[^,]+"\).)/ require("@tbcm\/common")./g' ./apps/api/dist/**/*.controller.js; \
+	fi
```

Comment:
What is the context behind this script?

Reply:
This fixes the Runtime Module error. App can't convert commons package relative path. Lines above these convert *.entity.js files, while this change takes care of *.controller.js.

The idea or enhancement later can be to take care of all src files. But do not want to touch a lot at a stretch at the moment.